src/App.css

```diff
@@ -1,42 +1,40 @@
 .App {
+  display: flex;
+  flex-direction: column;
+  gap: 25px;
+  align-items: center;
   text-align: left;
-}
-
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
 }
 
 .App-header {
-  background-color: #222;
-  height: 50px;
-  padding: 20px;
+  display: flex;
+  flex-direction: row;
```

Comment:
The default flex direction is row, you shouldn't need to add this.

Reply:
Thanks! Corrected.